modules/ossm-rn-new-features.adoc

```diff
@@ -23,18 +23,22 @@ Result – If changed, describe the current user experience
 |Component |Version
 
 |Istio
-|1.4.8
+|1.4.10
 
 |Jaeger
 |1.17.4
 
 |Kiali
-|1.12.7
+|1.12.15
 
 |3scale Istio Adapter
 |1.0.0
 |===
 
+== New features {ProductName} 1.1.8
+
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes. This release upgrades Istio to version 1.4.10.
```

Comment:
Minor thing, but perhaps link to the Istio 1.4.10 release notes: https://istio.io/latest/news/releases/1.4.x/announcing-1.4.10/ >